src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDirectiveDescriptor.cs

```diff
@@ -11,23 +11,24 @@ namespace Microsoft.AspNet.Razor.TagHelpers
     public class TagHelperDirectiveDescriptor
     {
         // Internal for testing purposes.
-        internal TagHelperDirectiveDescriptor(string lookupText,
+        internal TagHelperDirectiveDescriptor(string directiveText,
```

Comment:
existing class is well-named for existing purpose.  why extend it to support this new, quite different directive?